examples/java/pom.xml

```diff
@@ -138,12 +137,17 @@
                   <goal>verify</goal>
                 </goals>
                 <configuration>
+                  <includes>
+                    <include>WordCountIT.java</include>
```

Comment:
Yes, I tested it in my own directory. Both "WordCountIT.java" and "**/WordCountIT.java" works.